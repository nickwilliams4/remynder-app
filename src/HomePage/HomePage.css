
h1 {
  border-style: double;
  font-size: 50px;
}

h3 {
  border-style: outset;
}

h2 {
  font-style: italic;
}