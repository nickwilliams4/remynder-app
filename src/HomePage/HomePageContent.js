import React, { Component } from 'react'
import Content from '../Content/Content'

export default class HomePageContent extends Component {
  render() {
    return (
      <Content className='HomePageContent'>
        <header role="banner">
          <h1>Remynder</h1>
          <h2>get things done.</h2>
        </header>
        <section>
          <header>
            <h3>Organize tasks</h3>
          </header>
          <p>[<em>placeholder for screenshot of list of notes and/or folders of notes</em>]</p>
          <p>Remynder helps you create notes or tasks to be completed.</p>
        </section>
        <section>
          <header>
            <h3>Get remynders</h3>
          </header>
          <p>[<em>placeholder for screenshot of setting frequency of reminder notifications</em>]</p>
          <p>What sets this app apart is the ability to receive an email of your note or task directly to your inbox after creating a remynder, and how often you want to to receive email remynders after that.</p>
        </section>
        <section>
          <header>
            <h3>Keep remynders coming</h3>
          </header>
          <p>[<em>placeholder for screenshot of emailed reminders</em>]</p>
          <p>This application will make you more productive by sending remynders to your inbox as often as you would like until you complete your task. Checking our email often is something we are all accustomed to doing so give yourself an edge to get things done.</p>
        </section>
        <section>
          <header>
            <h3>Start maximizing your productivity now!</h3>
          </header>
          </section>
      </Content>
    )
  }
}