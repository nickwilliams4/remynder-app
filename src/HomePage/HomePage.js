import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import Content from '../Content/Content'
import './HomePage.css'

export default class HomePage extends Component {
  render() {
    return (
      <Content className='HomePage'>
        <main role="main">
          <header role="banner">
            <h1>Remynder</h1>
            <h2>get things done.</h2>
          </header>
          <section>
            <header>
              <h3>Organize tasks</h3>
            </header>
            <p>[<em>placeholder for screenshot of list of notes and/or folders of notes</em>]</p>
            <p>Remynder helps you create notes or tasks to be completed.</p>
          </section>
          <section>
            <header>
              <h3>Get remynders</h3>
            </header>
            <p>[<em>placeholder for screenshot of setting frequency of reminder notifications</em>]</p>
            <p>What sets this app apart is the ability to receive an email of your note or task directly to your inbox after creating a remynder, and how often you want to to receive email remynders after that.</p>
          </section>
          <section>
            <header>
              <h3>Keep remynders coming</h3>
            </header>
            <p>[<em>placeholder for screenshot of emailed reminders</em>]</p>
            <p>If you are like most of us, you are constantly checking your inbox. This application will make you more productive by sending your remynders to a spot you are already checking many times a day.</p>
          </section>
          <section>
            <header>
              <h3>Up Your Production Now</h3>
            </header>
            <form class='signup-form'>
              <div>
                <label for="first-name">First name: </label>
                <input placeholder='First Name' type="text" name='first-name' id='first-name' />
              </div>
              <div>
                <label for="last-name">Last name: </label>
                <input type="text" name='last-name' id='last-name' placeholder='Last Name' />
              </div>
              <div>
                <label for="username">Email: </label>
                <input type="text" name='username' id='username' placeholder="john@lennon.com" />
              </div>
              <div>
                <label for="password">Password: </label>
                <input type="password" name='password' id='password' />
              </div>
              <Link to='/NotePage'>
                <button type='submit'>Sign Up</button>
              </Link>
            </form>
          </section>
        </main>
      </Content>
    )
  }
}