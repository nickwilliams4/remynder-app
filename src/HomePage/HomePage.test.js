import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HomePage from './HomePage'

describe('HomePage component', () => {
  const props = {
    className: 'test-class-name'
  }

  it('renders the HomePage given props', () => {
    const wrapper = shallow(<HomePage {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})