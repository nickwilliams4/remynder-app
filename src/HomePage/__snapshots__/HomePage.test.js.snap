// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders the HomePage given props 1`] = `
<Content
  className="HomePage"
>
  <main
    role="main"
  >
    <HomePageContent />
    <Link
      to="/SignUpForm"
    >
      <button
        type="submit"
      >
        Sign Up!
      </button>
    </Link>
  </main>
</Content>
`;
